src/platform/linux/x11/xdisplay.rs

```diff
@@ -10,7 +10,8 @@ use super::ffi;
 /// A connection to an X server.
 pub struct XConnection {
     pub xlib: ffi::Xlib,
-    pub xrandr: ffi::Xrandr,
+    pub xrandr: ffi::Xrandr_2_2_0,
+    pub xrandr_1_5: Option<ffi::Xrandr>,
```

Comment:
Could you add some comment specifying that the first member concerns XRandr < 1.5 and the second one concerns XRandr >= 1.5? The names and types are a bit confusing otherwise.